cluster/gke/config-test.sh

```diff
@@ -18,11 +18,22 @@
 CLUSTER_NAME="${CLUSTER_NAME:-${USER}-gke-e2e}"
 NETWORK=${KUBE_GKE_NETWORK:-e2e}
 NODE_TAG="k8s-${CLUSTER_NAME}-node"
-IMAGE_TYPE="${KUBE_GKE_IMAGE_TYPE:-container_vm}"
+IMAGE_TYPE="${KUBE_GKE_IMAGE_TYPE:-cos}"
```

Comment:
No, it seems to be relevant only within cluster/gke.  That said, IMAGE_TYPE can be set when running kube-up.sh to indicate which image to deploy (cos||container_vm), so it has some value in indicating what was _last_ deployed.

I don't really like using it to detect image though.  If I create 2 GKE clusters, 1 cos and 1 container_vm, and switch between them for testing, IMAGE_TYPE won't mirror the latest cluster context.